unkeyed/unkey
# feat(worker): added opentelemetry

/claim #975
/fixes #975

## What does this PR do?

Integrated opentelemetry.

fixes #975 

## Type of change


## How should this be tested?

This PR is completely tested. You just need to insert your baselime api key and service name in .dev.vars file. After doing this make some api calls like verifying keys. And finally after some time [15-20 mins], the traces will be visible on baselime console.

`pnpm fmt` is showing some warnings in the files that I have not made any changes to.


## Checklist

### Required


### Appreciated
